### PR TITLE
Capitalize acronyms and abbreviations

### PR DESCRIPTION
**Pull request by hpohekar**
* Capitalizing acronyms and abbreviations

### CODE REVIEW

File: src/ansys/fluent/core/services/datamodel_se.py [hunk 1]
```diff
@@ -89,63 +89,63 @@ def __init__(
     def initialize_datamodel(
         self, request: DataModelProtoModule.InitDatamodelRequest
     ) -> DataModelProtoModule.InitDatamodelResponse:
-        """initDatamodel rpc of DataModel service."""
+        """InitDatamodel rpc of DataModel service."""
```
Comment (mkundu1) [inline src/ansys/fluent/core/services/datamodel_se.py:92]: Is this changed by pre-commit? I added these words in an exclusion list for the docformatter, probably some local sync issue? If I run `pre-commit run --all-files` locally it doesn't report any formatting change.

File: src/ansys/fluent/core/post_objects/post_object_definitions.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def display(self, window_id: Optional[str] = None):
         Parameters
         ----------
         window_id : str, optional
-            Window id. If not specified unique id is used.
+            Window id. If not specified unique ID is used.
```
Comment (raph-luc) [inline src/ansys/fluent/core/post_objects/post_object_definitions.py:46]:
```suggestion
            Window ID. If not specified a unique ID is used.
```

File: src/ansys/fluent/core/post_objects/post_object_definitions.py [hunk 3]
```diff
@@ -58,7 +58,7 @@ def plot(self, window_id: Optional[str] = None):
...
```
Comment (raph-luc) [inline src/ansys/fluent/core/post_objects/post_object_definitions.py:61]:
```suggestion
            Window ID. If not specified a unique ID is used.
```

File: src/ansys/fluent/core/services/field_data.py [hunk 4]
```diff
@@ -740,7 +740,7 @@ def _get_surface_ids(
     surface_names: Optional[List[str]] = None,
     surface_name: Optional[str] = None,
 ) -> List[int]:
-    """Get surface ids' based on surface names or ids'.
+    """Get surface IDs' based on surface names or IDs'.
```
Comment (raph-luc) [inline src/ansys/fluent/core/services/field_data.py:743]:
```suggestion
    """Get surface IDs based on surface names or IDs.
```

**Comment by mkundu1** [comment]
Can we enforce this through vale?

File: src/ansys/fluent/core/services/datamodel_se.py [hunk 5]
```diff
@@ -89,63 +89,63 @@ def __init__(
...
         return self._stub.initDatamodel(request, metadata=self._metadata)
 
     @catch_grpc_error
     def get_attribute_value(
         self, request: DataModelProtoModule.GetAttributeValueRequest
     ) -> DataModelProtoModule.GetAttributeValueResponse:
-        """getAttributeValue rpc of DataModel service."""
+        """GetAttributeValue rpc of DataModel service."""
```
Comment (raph-luc) [inline src/ansys/fluent/core/services/datamodel_se.py:99]:
```suggestion
        """GetAttributeValue RPC of DataModel service."""
```
RPC also should be capitalized

Are there any other acronyms/abbreviations that we may be missing?

There are a few places in docstrings with "gprc", should be capitalized as gRPC: https://grpc.io/

File: src/ansys/fluent/core/services/datamodel_se.py (hunk 1 above)
Comment (hpohekar) [reply src/ansys/fluent/core/services/datamodel_se.py:92]: Is this changed by pre-commit? - yes.

`pre-commit run --all-files` does not resolve this issue.

File: doc/source/conf.py [hunk 6]
```diff
@@ -142,7 +142,7 @@ def _stop_fluent_container(gallery_conf, fname):
     "examples_dirs": ["../../examples/"],
     # path where to save gallery generated examples
     "gallery_dirs": ["examples"],
-    # Pattern to search for example files
+    # Pattern to search for example, files
```
Comment (raph-luc) [inline doc/source/conf.py:145]:
```suggestion
    # Pattern to search for example files
```

**Review by raph-luc — APPROVED** [review]

File: src/ansys/fluent/core/post_objects/post_object_definitions.py (hunk 2 above)
Comment (hpohekar) [reply src/ansys/fluent/core/post_objects/post_object_definitions.py]: @raph-luc  I missed `If not specified,` Done. Thank you.

File: doc/source/conf.py (hunk 6 above)
Comment (hpohekar) [reply doc/source/conf.py]: Done. Thank you.

**Comment by seanpearsonuk** [comment]
It hadn't occurred to me about e.g. :) but your changes are good, thanks

**Review by seanpearsonuk — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**